dsgrid/registry/common.py

```diff
@@ -62,6 +62,7 @@ class ProjectRegistryStatus(DSGEnum):
     INITIAL_REGISTRATION = "Initial Registration"
     IN_PROGRESS = "In Progress"
     COMPLETE = "Complete"
+    PUBLISHED = "Published"
```

Comment:
@mooneyme @elainethale Is this the right term? Meghan used PUBLISHED-INTERNAL or PUBLISHED-OEDI in Jira DSGRID-45.

Reply:
I *think* the type of publication will be indicated by the dsgrid registry location? That is, where we are currently developing is effectively internal and when we publish on OEDI we'll need to make a new dsgrid registry there? If that is the correct interpretation, than I think "Published" is sufficient--the type of publication is indicated by the registry location and who can get to it.